### ISSUE TITLE
inconsistency at count property

### ISSUE THREAD

**Issue by alejovdev** [zandoan/turing-frontend#18]
When i made the request to the /products, the count is just the number, but on the /products/inCategory/x or /products/inDepartment/x is an an object containing another extra property of count with the actual number.

![image](https://user-images.githubusercontent.com/37085202/55771120-574a1d00-5a4c-11e9-8cc2-3a649dbb7640.png)

![image](https://user-images.githubusercontent.com/37085202/55771142-616c1b80-5a4c-11e9-99e1-d2baa2b48685.png)

**Comment by edertaveiraturing** [comment]
Yes. We are fixing it asap. So when I have news about it I'll comment here. Thank you.
